mm2src/coins/tendermint/tendermint_tx_history_v2.rs

```diff
@@ -145,8 +145,9 @@ impl CoinWithTxHistoryV2 for TendermintToken {
         &self,
         _target: MyTxHistoryTarget,
     ) -> MmResult<GetTxHistoryFilters, MyTxHistoryErrorV2> {
-        let denom_hash = sha256(self.denom.to_string().as_bytes());
-        let token_id = H256::from(denom_hash.take()).to_string();
+        let denom_hash = sha256(self.denom.as_ref().to_lowercase().as_bytes());
+        let bytes: BytesJson = H256::from(denom_hash.take()).to_vec().into();
+        let token_id = format!("{:02x}", bytes);
```

Comment:
doesn't `H256.to_string()` return it as a hex string correclty?

Reply:
It didn't, I can re-check to confirm.